lib/src/react16_suggestors/react_dom_render_migrator.dart

```diff
@@ -70,15 +88,24 @@ class ReactDomRenderMigrator extends GeneralizingAstVisitor
       return ancestor is FunctionDeclaration;
     });
 
-    void addOverReactPatch(int offset) {
-      yieldPatch(
-          offset, offset, 'import \'package:over_react/over_react.dart\';\n');
+    void addOverReactPatch([int offset]) {
+      if (offset == null && imports.isNotEmpty) {
+        offset = imports.last.offset;
+      } else if (libraryDirective.isNotEmpty) {
+        offset = libraryDirective.first.end;
+      } else {
+        offset = 0;
+      }
+      if (offset != null && !isPartOf) {
```

Comment:
#nit because offset will always be set now, I don't believe we need to check that it's null

```suggestion
      if (!isPartOf) {
```

Reply:
good point